src/pages/_document.js

```diff
@@ -20,6 +20,7 @@ export default class MyDocument extends Document {
           <link rel="icon" type="image/png" sizes="32x32" href="/assets/favicon/favicon-32x32.png" />
           <link rel="icon" type="image/png" sizes="16x16" href="/assets/favicon/favicon-16x16.png" />
           <link rel="manifest" href="/assets/favicon/site.webmanifest" />
+          <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
```

Comment:
Rather than importing the entire CSS file, let's use the corresponding npm package so we can select the exact icons we want to use

Reply:
i have to because i installed the NPM package, the library is not as updated as this one

Reply:
they are maintained and released at the same versions - https://www.npmjs.com/package/font-awesome

let's use the npm package for this instead